src/views/scroll-editor/manuscript-toolbar.vue

```diff
@@ -500,7 +528,63 @@ export default class ManuscriptToolbar extends Vue {
         this.params.mode = mode;
     }
 
+    private getArtefactHeight(artefact?: Artefact) {
+        if (artefact) {
```

Comment:
Why `arefact?` and the test for artefact? Also, there's no return type annotation which is a shame.